staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go

```diff
@@ -42,25 +44,92 @@ type apisHandler struct {
 	discoveryGroup metav1.APIGroup
 }
 
-func discoveryGroup(enabledVersions sets.String) metav1.APIGroup {
+func getGroupVersionHash(path string, handler http.Handler) (string, error) {
```

Comment:
Same comment as elsewhere in this review that there is now a library function of this